### PR TITLE
Added Quickstarts for Custom Repos, Launch AWS, Launch Azure.

### PR DESCRIPTION
**Pull request by arburka**
Adding Quickstarts for the three Image quickstarts
- Custom Repos
- AWS Launch
- Azure Launch

### CODE REVIEW

**Comment by resoluteCoder** [comment]
@Hyperkid123 Per our previous conversation we are updating the docs with quickstarts. Hopefully everything is in the right place and has the specific values. One I could think of that may be incorrect is in the `metadata.yml` file specifically the value `image builder`.

thanks again for all your guidance and help!! :smile:

File: docs/quickstarts/insights-custom-repos/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+kind: QuickStarts # kind must always be "QuickStarts"
+name: insights-custom-repos
+tags: # If you want to use more granular filtering add tags to the quickstart
+  - kind: bundle
+    value: insights
+  - kind: application # use application tag for quickstart used by specific application
+    value: image builder
```
Comment (Hyperkid123) [inline docs/quickstarts/insights-custom-repos/metadata.yml:7]: This should match the application UI module ID. For image builder it is `imageBuilder`

Comment (arburka) [reply docs/quickstarts/insights-custom-repos/metadata.yml]: Thank you! I made the updates @Hyperkid123 !

**Comment by shaded-enmity** [comment]
@Hyperkid123 hey Martin, can you please take a look if everything is good from your perspective?

File: docs/quickstarts/insights-custom-repos/insights-custom-repos.yml [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+          
+              - Click **Next.**
+                    
+          1. On the “Target Environment” Steps:
+              
+              - Enter additional information based on the target environment selected.
+                  
+                  - **AWS:** Choose one of the sharing methods:
+                      
+                      - Use the “Use an account configured from [Sources] (https://gitlab.cee.redhat.com/red-hat-enterprise-linux-documentation/host-management-services/-/blob/main/modules/provisioning/proc_connecting-aws-account-with-red-hat-hybrid-cloud-console.adoc)” option and choose the recently created or edited AWS source.
```
Comment (shaded-enmity) [inline docs/quickstarts/insights-custom-repos/insights-custom-repos.yml:102]: Replace link with https://console.redhat.com/beta/settings/sources

File: docs/quickstarts/insights-custom-repos/insights-custom-repos.yml [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    
+    <br>
+    **<a id="qs-ib-custom-content-contact" href="">Let Red Hat to contact you <i class="fas fa-arrow-right"></i> </a>**
+    <br>
+    <br>
+    <br>
+    
+    If you need additional assistance:
+      
+      - Learn more about [custom repositories](https://gitlab.cee.redhat.com/red-hat-enterprise-linux-documentation/host-management-services/-/blob/main/modules/content/proc_adding-custom-repository.adoc) in our documentation.
```
Comment (shaded-enmity) [inline docs/quickstarts/insights-custom-repos/insights-custom-repos.yml:199]: Remove this link, we'll it back once docs are merged

File: docs/quickstarts/insights-launch-aws/insights-launch-aws.yml [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+  description: |-
+    Learn how to add an Amazon Web Service source and launch an image.
+  introduction: |-
+    Welcome to the quick start for launching an Amazon Web Service (AWS) image.
+
+    In this quick start, you learn how to add or edit &nbsp;[AWS sources] (https://gitlab.cee.redhat.com/red-hat-enterprise-linux-documentation/host-management-services/-/blob/main/modules/provisioning/proc_connecting-aws-account-with-red-hat-hybrid-cloud-console.adoc), create an image, and launch it all on the Red Hat Hybrid Cloud Console.
```
Comment (shaded-enmity) [inline docs/quickstarts/insights-launch-aws/insights-launch-aws.yml:26]: Replace link with https://console.redhat.com/beta/settings/sources

File: docs/quickstarts/insights-launch-aws/insights-launch-aws.yml [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+    **<a id="qs-ib-aws-launch-contact" href="">Let Red Hat to contact you <i class="fas fa-arrow-right"></i> </a>**
+    <br>
+    <br>
+    <br>
+   
+    If you need additional assistance:
+      
+      - Learn more about [Sources](https://www.example.com)in our documentation.
+
+      - Learn more about [Launching](https://www.example.com)in our documentation.
```
Comment (shaded-enmity) [inline docs/quickstarts/insights-launch-aws/insights-launch-aws.yml:302]: Remove these, we'll add them back once these exist

File: docs/quickstarts/insights-launch-azure/insights-launch-azure.yml [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+    **<a id="qs-ib-azure-launch-contact" href="">Let Red Hat to contact you <i class="fas fa-arrow-right"></i> </a>**
+    <br>
+    <br>
+    <br>
+   
+    If you need additional assistance:
+      
+      - [Sources documentation](https://www.example.com)
+
+      - [Launching documentation](https://www.example.com)
```
Comment (shaded-enmity) [inline docs/quickstarts/insights-launch-azure/insights-launch-azure.yml:297]: Ditto from above

**Comment by Hyperkid123** [comment]
@arburka I have tested the content locally with the quickstart API updates and it is working. Are we good to merge or do you want to make some additional changes?

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**